Progressus-io/postnl-for-woocommerce
# Fix HPOS Path Declaration

Fix HPOS path declaration by changing the string value into `POSTNL_WC_PLUGIN_FILE` constant variable.


### Steps to test : 
1. Activate the plugin.
2. Make sure no error is listed in the log.
3. And try to enable the HPOS feature from **WooCommerce >> Settings >> Advanced >> Features**.
4. Make sure there is no WooCommerce notice is displayed.
